### PR TITLE
Applying helm3 branch

### PR DESCRIPTION
**Pull request by tomahawk28**
Toward https://github.com/giantswarm/giantswarm/issues/10973

Applying `helm3` branches into master. Added Changelog entry. 

## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changed
 
+- Changed to support Helm 3; To keep using Helm 2, please use the version under 0.2.2.
```
Comment (tomahawk28) [inline CHANGELOG.md:12]: Is this should be added or changed? Since this is break changes, I prefer to have it as changed.

File: README.md [hunk 2]
```diff
@@ -3,22 +3,16 @@
...
-a port forwarding connection.
+releases. Currently supports Helm 3.
 
 ## Interface
 
 See `helmclient.Interface` in [spec.go] for supported methods.
 
-## Related libraries
-
-[k8sportforward] is used to establish a port forwarding conection with Tiller.
```
Comment (tomahawk28) [inline README.md:15]: k8sportforward was deleted.

File: CHANGELOG.md (hunk 1 above)
Comment (rossf7) [reply CHANGELOG.md:12]:
```suggestion
- Updated to support Helm 3; To keep using Helm 2, please use version 0.2.X.
```

Good point. What about this?  If we have to make fixes for Helm 2 the version will be 0.2.X.

Comment (tomahawk28) [reply CHANGELOG.md:12]: Thanks, that makes sense.

**Pull request merged**